public/public.php

```diff
@@ -644,7 +644,7 @@ public function list_all_post_attachments_sources_shortcode( $atts = [] ) {
 			$connected_atts[ $_attachment->ID ]['source']   = self::get_image_source_text( $_attachment->ID );
 			$connected_atts[ $_attachment->ID ]['standard'] = Standard_Source::use_standard_source( $_attachment->ID );
 			// jump to next element if the standard source is set to be excluded from the source list
-			if ( Standard_Source::standard_source_is( 'exclude' ) && '' !== $connected_atts[ $_attachment->ID ]['standard'] ) {
+			if ( Standard_Source::standard_source_is( 'exclude' ) && $connected_atts[ $_attachment->ID ]['standard'] ) {
 				unset( $connected_atts[ $_attachment->ID ] );
 				continue;
```

Comment:
**Suggestion:** The variable `$_attachment` in line 647 is not very descriptive. Consider renaming it to something more meaningful. [best practice]
```suggestion
   if ( Standard_Source::standard_source_is( 'exclude' ) && $connected_atts[ $attachment_id ]['standard'] ) {
     unset( $connected_atts[ $attachment_id ] );
     continue;
   }
```